src/cpp/tests/testthat/themes/.gitignore

```diff
@@ -0,0 +1 @@
+temp
```

Comment:
Is this .gitignore change related to this or just a stray?

Reply:
Sorry, it's just piggybacking on this commit. I noticed the temp folder was created while tests were running and wanted to avoid accidentally committing the folder.